### PR TITLE
Use `property` attribute for OpenGraph title and image

### LINKED ISSUE #3211: `name` incorrectly used instead of `property` for OpenGraph tags
## Description

OpenGraph tags for title, description, and image all incorrectly use a `name` instead of `property` attribute.

`<meta name="og:title" content="A super cool bird">` should be `<meta property="og:title" content="A super cool bird">`

To fix this, update the `createDetailPageMeta` function to use `property` instead of `name`: https://github.com/WordPress/openverse/blob/aa3a7f5b37f5baf590967b950aade0c18299e3cf/frontend/src/utils/og.ts

## Additional context

OpenGraph Protocol definition: https://ogp.me/#metadata

### PR DESCRIPTION
**Pull request by RogueCode007**
## Fixes

Fixes #3211 by @sarayourfriend 

## Description
Change OpenGraph tags for title, description, and image from using a name attribute to using a property attribute instead.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: frontend/src/utils/og.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import type { MetaInfo } from "vue-meta"
-import type { MetaPropertyName } from "vue-meta/types/vue-meta"
+import type { MetaPropertyProperty } from "vue-meta/types/vue-meta"
```
Comment (dhruvkb) [inline frontend/src/utils/og.ts:2]: These types are imported from the `vue-meta` package and must use the name set by that package. Please revert this change.

File: frontend/src/utils/og.ts [hunk 2]
```diff
@@ -15,21 +15,21 @@ export const createDetailPageMeta = ({
   const meta = [
     {
       hid: "robots",
-      name: "robots",
+      property: "robots",
```
Comment (dhruvkb) [inline frontend/src/utils/og.ts:18]: The [robots meta tag](https://developers.google.com/search/docs/crawling-indexing/robots-meta-tag#robotsmeta) must use `name` and `content`. Please revert this change.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Thanks for contributing to the issue @RogueCode007! There are some changes required to make the code work properly. Additionally, after making these changes, you will need to update the tests.

**Review by obulat — COMMENTED** [review]
@RogueCode007, `og.ts` only creates the meta for single result pages (single image and single audio).
To change the `meta` for other pages, you'll need to update these lines:
https://github.com/WordPress/openverse/blob/0d95b6213472be1a737e2bf3bee211c582457b78/frontend/src/constants/meta.ts#L27-L31

**Comment by RogueCode007** [comment]
> @RogueCode007, `og.ts` only creates the meta for single result pages (single image and single audio). To change the `meta` for other pages, you'll need to update these lines:
> 
> https://github.com/WordPress/openverse/blob/0d95b6213472be1a737e2bf3bee211c582457b78/frontend/src/constants/meta.ts#L27-L31

ah thanks! i was wondering what was wrong 

> @RogueCode007, `og.ts` only creates the meta for single result pages (single image and single audio). To change the `meta` for other pages, you'll need to update these lines:
> 
> https://github.com/WordPress/openverse/blob/0d95b6213472be1a737e2bf3bee211c582457b78/frontend/src/constants/meta.ts#L27-L31

**Review by obulat — APPROVED** [review]
Awesome work, @RogueCode007 ! Thank you for your contribution 🎉

**Review by dhruvkb — APPROVED** [review]
All good, LGTM. Tested locally and everything works. Thanks for the contribution @RogueCode007!

**Pull request merged**